hudi-common/src/test/java/org/apache/hudi/common/testutils/FileCreateUtils.java

```diff
@@ -83,6 +83,17 @@ private static void createMetaFile(String basePath, String instantTime, String s
     }
   }
 
+  private static void createMetaFile(FileSystem fs, String basePath, String instantTime, String suffix) throws IOException {
```

Comment:
dont we already have helpers that do these. cc @xushiyan

Reply:
`HoodieTestTable` makes use of this `FileCreateUtils` to perform the actual file manipulations.

Reply:
Looks like this new set of helpers is a version that uses hadoop fs instead of java nio. Would it align better if we put `FileSystem fs` as the last argument? so that other args match with the java nio version. This applies to the few new helper methods below.

Reply:
+1